libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -439,6 +439,9 @@ uint8_t WiFiClient::connected()
     if (_connected) {
         uint8_t dummy;
         int res = recv(fd(), &dummy, 0, MSG_DONTWAIT);
+        if(res < 0) {
+            log_e("%d", errno);
```

Comment:
logging the errno is not meaningful without a message to say why it is being logged. It would also be beneficial to log the "res" value here possibly.

Reply:
Yes you are right should be res and not errno - will update it

Reply:
changed to : ` log_e("RES: %d", res);`